flocker/provision/test/test_digitalocean.py

```diff
@@ -0,0 +1,67 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.provision._digitalocean``.
+"""
+import copy
+import os
+
+import pyocean
+
+from twisted.trial.unittest import SynchronousTestCase, SkipTest
+
+from flocker.provision._digitalocean import (
+    set_latest_droplet_kernel, retry_if_pending)
+from flocker.testtools import random_name
+
+
+TESTING_DROPLET_ATTRIBUTES = {
+    'region': 'lon1',
+    'size': '8gb',
```

Comment:
It's the spec that we use in the installation documentation:
 * http://doc-dev.clusterhq.com/gettingstarted/installation.html#using-digitalocean